plink-front/src/app/components/coin/coin.component.scss

```diff
@@ -0,0 +1,67 @@
+@import '~src/scss/_colors';
+
+.coin-container {
+  border: 1px solid $gray;
+  display: grid;
+  grid-template-columns: 30% 25% 25% 20%;
```

Comment:
since it's a single row you could use flex instead

Reply:
I'd also set `flex: 3` in the first element, and `flex: 1` in the second and third leaving the button to wrap itself